tsp-output/@azure-tools/typespec-csharp/src/Generated/Models/CreateChatCompletionRequest.Serialization.cs

```diff
@@ -268,6 +268,29 @@ void IJsonModel<CreateChatCompletionRequest>.Write(Utf8JsonWriter writer, ModelR
 #endif
                 }
             }
+
+            // Note: we were holding strongly-typed values for added Azure properties
+            // We have to serialize them separately here.
+            // TODO: Does the format matter here?  It seems like we want to write them
```

Comment:
Look at this in the context of different options for Wire-3rd party and Wire-Azure.